esiclient/v1/node_network.py

```diff
@@ -92,15 +97,26 @@ def take_action(self, parsed_args):
                 if not filter_network or filter_network.id == network_id:
                     network_names, _, fixed_ips \
                         = utils.get_full_network_info_from_port(
-                            neutron_port, neutron_client)
+                            neutron_port, neutron_client, networks_dict)
+                    floating_ip_addresses, floating_network_names \
+                        = utils.get_floating_ip(neutron_port_id,
+                                                floating_ips,
+                                                networks_dict)
                     data.append([node_name, port.address,
                                  neutron_port.name,
                                  "\n".join(network_names),
-                                 "\n".join(fixed_ips)])
+                                 "\n".join(fixed_ips),
+                                 "\n".join(floating_ip_addresses)
+                                 if floating_ip_addresses else None,
+                                 "\n".join(floating_network_names)
+                                 if floating_network_names else None]
+                                )
             elif not filter_network:
-                data.append([node_name, port.address, None, None, None])
+                data.append([node_name, port.address,
+                             None, None, None, None, None])
 
-        return ["Node", "MAC Address", "Port", "Network", "Fixed IP"], data
+        return ["Node", "MAC Address", "Port", "Network", "Fixed IP",
+                "Floating IP", "Floating Network"], data
```

Comment:
It may make sense to switch around "Floating IP" and "Floating Network" to match "Network" and "Fixed IP"

Reply:
Fixed!